regression_tests/ref_counter.jl

```diff
@@ -1 +1 @@
-31
+32
```

Comment:
If we are bumping the ref counter we don't need to also change the mase tables?

Reply:
There's a new job which opts into the regression tests, so we do need to increment the ref counter